config/webpack.config.js

```diff
@@ -52,11 +65,22 @@ var config = {
       path.join(__dirname, '..', 'webpack'),
       'node_modules/'
     ],
-    alias: {
+    alias: Object.assign({
       foremanReact:
         path.join(__dirname,
-           '../webpack/assets/javascripts/react_app/components')
-    }
+           '../webpack/assets/javascripts/react_app/components'),
+      foremanReactCommon:
+        path.join(__dirname,
+          '../webpack/assets/javascripts/react_app/common'),
+      foremanReactService:
+        path.join(__dirname,
+          '../webpack/assets/javascripts/services'),
+      foremanNodeModules:
```

Comment:
does it mean that all plugins need to prefix their requirements to foreman node module? that would explain why in my tests i had a very large vendor file and a plugin specific file...I could see this being fairly annoying + hard to use npm packages that include common other libs (e.g. a react-date picker or something like that)